<?xml version="1.0"?>
<launch>
  <!-- configs -->
  <param name="use_sim_time" value="false" />
  <param name="sim_mode" value="false" />
  <param name="debug_mode" value="false" />
  <param name="camera_config_dir" value="$(find awesomo_configs)/configs/camera/ximea_camera" />
  <param name="grey_scale" type="bool" value="true" />

  <!-- ros node -->
  <node pkg="awesomo_ros" name="awesomo_camera" type="ximea_camera_node" output="screen" required="true" />
</launch>
